ni_measurementlink_service/session_management.py

```diff
@@ -418,6 +444,520 @@ def create_sessions(
         """
         return self._create_sessions_core(session_constructor, instrument_type_id)
 
+    @requires_feature(SESSION_MANAGEMENT_2024Q1)
+    def create_nidcpower_session(
+        self,
+        reset: bool = False,
+        options: Optional[Dict[str, Any]] = None,
+        initialization_behavior: SessionInitializationBehavior = SessionInitializationBehavior.AUTO,
+    ) -> ContextManager[TypedSessionInformation[nidcpower.Session]]:
+        """Create a single NI-DCPower instrument session.
+
+        Args:
+            reset: Specifies whether to reset channel(s) during the
+                initialization procedure.
+
+            options: Specifies the initial value of certain properties for the
+                session. If this argument is not specified, the default value is
+                an empty dict, which you may override by specifying
+                ``NIDCPOWER_SIMULATE``, ``NIDCPOWER_BOARD_TYPE``, and
+                ``NIDCPOWER_MODEL`` in the configuration file (``.env``).
+
+            initialization_behavior: Specifies whether to initialize a new
+                session or attach to an existing session.
+
+        Returns:
+            A context manager that yields a session information object. The
+            created session is available via the ``session`` field.
+
+        See Also:
+            For more details, see :py:class:`nidcpower.Session`.
```

Comment:
https://github.com/ni/measurementlink-python/pull/430 turns this into a hyperlink.